src/drivers/mct/main/cesm_comp_mod.F90

```diff
@@ -4042,4 +4066,60 @@ subroutine cesm_comp_barriers(mpicom, timer)
   endif
 end subroutine cesm_comp_barriers
 
+subroutine cesm_cpl_init(comm, ninst, id)
+
+  !-----------------------------------------------------------------------
+  !
+  ! Initialize multiple coupler instances, if requested
+  !
+  !-----------------------------------------------------------------------
+
+  implicit none
+
+  integer , intent(inout) :: comm
+  integer , intent(out)   :: ninst
+  integer , intent(out)   :: id      ! instance ID, starts from 1
+  !
+  ! Local variables
+  !
+  integer :: ierr, inst_comm, mype, nu, numpes !, pes
+  integer :: cpl_ninst
+
+  namelist /cesm_cpl/ cpl_ninst
```

Comment:
I don't think we should be introducing "cesm" into any namelist or variable names. The namelist group could be multi_inst as a possibility.